integration/test_batch.py

```diff
@@ -51,9 +60,10 @@ def test_add_data_object(client: weaviate.Client, uuid: Optional[UUID], vector:
     )
 
 
-@pytest.mark.parametrize("from_object_uuid", [uuid.uuid4(), uuid.uuid4().hex, str(uuid.uuid4())])
-@pytest.mark.parametrize("to_object_uuid", [uuid.uuid4(), uuid.uuid4().hex, str(uuid.uuid4())])
-@pytest.mark.parametrize("to_object_class_name", [None, "Test2"])
+@pytest.mark.order(2)
```

Comment:
you could just add one entry in the fixture or at the beginning of the test.

 Otherwise it would be necessary to run the tests in order when debugging something (Also not totally sure if the delete_all() in the fixture would not remove the objects from the first test.

Reply:
I added the order because I want to have some object in Weaviate when creating the cross-refs. If the object does not exist it will fail, that is why I first do the objects creation then the references. The fixture is per module and the tests are passing so I assume it is called only once. I might be wrong.

My intention is to add data to the Batch but all to create them to see if what is added works.

We could also just test the `add_*` and add better tests later, where we also get the object and see if it is correct.

What do you say?

Reply:
I think tests should always be self-contained. Otherwise reproducing problems becomes very annoying. Why not just add an object at the beginning of the ref test (can be directly without batching)?